include/igl/adjacency_list.cpp

```diff
@@ -12,7 +12,7 @@
 
 template <typename Index, typename IndexVector>
 IGL_INLINE void igl::adjacency_list(
-    const Eigen::MatrixBase<Index>  & F,
+    const Eigen::PlainObjectBase<Index>  & F,
```

Comment:
Why change this to `PlainObjectBase`? We should use `MatrixBase` for inputs.

Reply:
My bad, this is because the templates were written with PlainObjectBase. I will switch it back.